constants.py

```diff
@@ -0,0 +1,15 @@
+BASE_URL = 'https://www.ebay.co.uk/'
+
+
+MAIN_PAGE_TITLE = 'Electronics, Cars, Fashion, Collectibles, Coupons and More | eBay'
```

Comment:
Break this file down so that it makes sense like: 
```
# Search Page 
SEARCH_INPUT = 'Intel NUC'
SEARCH_BOX = '#gh-ac'
so on 

# Results Page 
RESULTS_PAGE_TITLE = '| eBay'
```